packages/components/app/styles/components/table.scss

```diff
@@ -199,10 +190,68 @@ $hds-table-cell-padding-tall: 22px 16px 21px 16px; // the 1px difference is to a
         border-bottom-left-radius: $hds-table-inner-border-radius;
       }
 
-      // A TH WILL NEVER BE LAST CHILD, ONLY FIRST CHILD
+      // a <th> will never be last child, only first child
       td:last-child {
         border-bottom-right-radius: $hds-table-inner-border-radius;
       }
     }
   }
+
+  .hds-table__th,
```

Comment:
Similarly to the cell that @andgen404 and I have been noticing, the way the padding and the spacing is handled is a bit different between Figma and code. I don't think that's a problem necessarily, but a small result is the border between columns as a pseudo element is 35px in height, in Figma it's 36 (I know, nit picky, we can decide whether this really matters).

Reply:
This is because how borders are applied in an HTML table. Essentially we have one border all around the entity `<table>` (which you don't have) and then other borders are applied to the table cells (`<th>/<td>`) in different ways. This leads to the `<th>` cells in the `<thead>` to optically be `49px` instead of `48px` (they're `48px` in height, but there's an extra border on the top, the table border). 
In order to achieve an exact match with Figma, the CSS code would be overly complicated for little gain. So we decided (time ago, when the `Hds::Table` was implemented, to keep things simpler for sake or maintainability).